test/filetype.vim

```diff
@@ -0,0 +1,22 @@
+scriptencoding utf-8
+
+" Please use nerd-font if you watch icon-font
+
+let s:suite = themis#suite('WebDevIconsGetFileTypeSymbol')
+let s:assert = themis#helper('assert')
+
+function! s:suite.FileNodesDefaultSymbol()
+  call s:assert.equals('', WebDevIconsGetFileTypeSymbol())
+endfunction
+
+function! s:suite.VimSymbol()
+  call s:assert.equals('', WebDevIconsGetFileTypeSymbol('.vimrc'))
+endfunction
+
+function! s:suite.LicenseSymbol()
+  call s:assert.equals('', WebDevIconsGetFileTypeSymbol('LICENSE'))
```

Comment:
Would all the `WebDevIconsGetFileTypeSymbol` asserts be separate functions or would they get grouped together? Maybe this is just a first step as you alluded to...

Reply:
These can be grouped or separated individually.